recipe/meta.yaml

```diff
@@ -16,13 +16,14 @@ build:
 
 requirements:
   host:
-    - python >=3.7,<3.10
+    - python >=3.10,<3.13
     - pip
   run:
-    - python >=3.7,<3.10
+    - python >=3.10,<3.13
     - setuptools
-    - openff-toolkit >=0.10.6, <0.11.0
-    - smirnoff-plugins >=0.0.3
+    - openff-toolkit >=0.11
```

Comment:
The only relevant toolkit change I could find was the periodic/non-periodic vdW methods change, even though this is a pretty old one

https://docs.openforcefield.org/projects/toolkit/en/stable/releasehistory.html#major-release-adding-support-for-proteins-and-refactoring-the-topology-class